src/pages/instances/InstanceDetailPanel.tsx

```diff
@@ -201,32 +201,53 @@ const InstanceDetailPanel: FC = () => {
                         <Link
                           to={`/ui/${instance.project}/instances/detail/${instance.name}/snapshots`}
                         >
-                          Recent snapshots
+                          Snapshots
                         </Link>
                       </h3>
                       {instance.snapshots?.length ? (
-                        <List
-                          items={instance.snapshots
-                            .reverse()
-                            .slice(0, RECENT_SNAPSHOT_LIMIT)
-                            .map((snapshot) => (
-                              <Row key={snapshot.name} className="no-grid-gap">
-                                <Col
-                                  size={4}
-                                  className="u-truncate"
-                                  title={snapshot.name}
+                        <>
+                          <List
```

Comment:
Also to align the width with elements above, I think we shouldn't use a list here (or the row / col below). Maybe use a tr for each snapshot to match the structure above.